components/LogoNav/LogoNav.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render, screen, fireEvent, waitFor } from '@testing-library/react';
+import LogoNav from './LogoNav';
+
+describe('LogoNav Component', () => {
+  it('renders the logo and links to the /league/all page', () => {
+    render(<LogoNav />);
+
+    const logoImage = screen.getByTestId('logo-nav');
+    expect(logoImage).toBeInTheDocument();
+
+    const linkElement = screen.getByRole('link', {
```

Comment:
Are we able to use the datatest-id of `logo-nav` here instead?

Reply:
The `data-testid="logo-nav"` is currently used for the Image tag. I like the idea of adding a data-testID to test the Link tag and will implement one shortly.